docs/faq/general/what-if-i-find-a-bug.md

```diff
@@ -0,0 +1,3 @@
+# What if I find a bug?
```

Comment:
The main reason was to easily track which questions people were clicking on - for something like the bug filing guide, it would achieve the same goal so maybe it does make sense to just link there directly.

Reply:
That's a fair point. Yeah, could be that it's just specifically the bug filing one that made me itchy.